docs/explanation/development-process.rst

```diff
@@ -312,9 +312,111 @@ the "Final Release", a representative of the team announces it on the
 Stable Release Updates
 ----------------------
 
-Released versions of Ubuntu are intended to be **stable**. This means that
-users should be able to rely on their behaviour remaining the same and
-therefore, updates are only released under particular circumstances.
-
-The dedicated :doc:`/explanation/stable-release-updates` article describes
-these criteria and the procedure for preparing such an update.
+After publication of a :term:`Ubuntu Stable Release`, there may be a need
+to update it or fix bugs. You can fix these newly discovered bugs and make
+updates through a special process known as **Stable Release Update (SRU)**.
+
+The SRU process ensures that any changes made to a stable release are thoroughly
+vetted and tested before being made available to users. This is because many of
+the users rely on the stability of the stable release for their day-to-day
```

Comment:
```suggestion
vetted and tested before being made available to users. This is because many
users rely on the stability of the stable release for their day-to-day
```